src/App.tsx

```diff
@@ -1,156 +1,101 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Selected } from './types/Selected';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { FilterTodos } from './components/FilterTodos';
+import { ErrorMessage } from './components/ErrorMessage';
+import { ErrorMessages } from './types/ErrorMessages';
+
+const USER_ID = 11281;
+
+function getPreparedTodos(
+  todos: Todo[], selected: Selected,
+) {
+  return todos.filter(todo => {
+    switch (selected) {
+      case Selected.ACTIVE: {
+        return !todo.completed;
+      }
+
+      case Selected.COMPLETED: {
+        return todo.completed;
+      }
+
+      default:
+        return todos;
+    }
+  });
```

Comment:
```suggestion
    switch (selected) {
      case Selected.ACTIVE: {
        return todos.filter(todo => !todo.completed);
      }

      case Selected.COMPLETED: {
        return todos.filter(todo => todo.completed);
      }

      default:
        return todos;
    }
```

for fix [this comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/742#discussion_r1291258535)

you can improve it in next tasks with todo_app